### PR TITLE
bump some version and fix some little things

### PR DESCRIPTION
**Pull request by polylymer**
### This pull request includes:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Breaking change (fix or feature that would cause existing api/functionality to change) --- https://github.com/Madfix-me/CloudNet-App/commit/05bc3cc1be5e2303178ea4a78266070d321ade92 ??
- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) document.
- [x] My code follows the code style of this project.
- [?] I have updated the documentation accordingly. ---  don't think there's anything which must be updated
- [x] I have added tests to cover my changes.  ---> tested on a physical android device

### CODE REVIEW

**Comment by TheMeinerLP** [comment]
Can your separate you dependency update into a different pull request?
Also I bumped in the branch `feature/task-edit-module-structure` already the flutter dependencies. 
Would be nice if it is possible.

Thanks for contributing and help us to get a UI/UX for cloudnet.

File: lib/utils/dialogs.dart [hunk 1]
```diff
@@ -260,7 +260,7 @@ StatefulBuilder addEditDeployment(
     ServiceDeployment deployment,
     NodeState state,
     void Function(ServiceDeployment deployment) save) {
-  var _excludeController = TextEditingController();
```
Comment (TheMeinerLP) [inline lib/utils/dialogs.dart:263]: Why not longer private ?
`_varname` means private

Comment (TheMeinerLP) [reply lib/utils/dialogs.dart:263]: I know, is here in the function unnecessary.

File: lib/feature/node/node_page.dart [hunk 2]
```diff
@@ -42,79 +42,81 @@ class _MenuNodePageState extends State<MenuNodePage> {
...
-                child: TextFormField(
-                  controller: _nameController,
-                  keyboardType: TextInputType.name,
-                  decoration: InputDecoration(
-                    hintText: t.page.menu_node.name,
-                    labelText: t.page.menu_node.node_name,
-                    suffixIcon: IconButton(
-                      icon: const Icon(Icons.clear),
-                      onPressed: () => _clearInputField(_nameController),
+        child: SingleChildScrollView(
```
Comment (TheMeinerLP) [inline lib/feature/node/node_page.dart:45]: Prevents overlapping at the bottom ?

Comment (polylymer) [reply lib/feature/node/node_page.dart:45]: yes, if you have the keyboard open and click on the advanced settings, there was a bottom overflow before that

File: lib/utils/dialogs.dart (hunk 1 above)
Comment (polylymer) [reply lib/utils/dialogs.dart]: https://github.com/Madfix-me/CloudNet-App/pull/28/commits/d9eab0221825521a009da6fc686279201875e9cb resolved this

File: lib/feature/node/node_page.dart (hunk 2 above)
Comment (TheMeinerLP) [reply lib/feature/node/node_page.dart:45]: Okay nice to know. You can resolve it or i am.

File: lib/apis/api_client.dart [hunk 3]
```diff
@@ -15,16 +15,13 @@ class ApiClient {
     dio = Dio()
       ..options.baseUrl = baseUrl
       ..interceptors.add(
-        InterceptorsWrapper(
+        QueuedInterceptorsWrapper(
```
Comment (TheMeinerLP) [inline lib/apis/api_client.dart:18]: Its based on this one right ?

https://github.com/flutterchina/dio/issues/1308

Comment (polylymer) [reply lib/apis/api_client.dart:18]: yes, but i don't quite understand the difference between the two classes, but without the deprecated methods it worked for me the same as before. I didn't find any good documentation about this change, especially how to handle .lock() and .unlock() after this change

Comment (TheMeinerLP) [reply lib/apis/api_client.dart:18]: Okay. This issue/pr link is for me enough. I ask back my developer in my company what the different is it.

**Pull request merged**